src/Symfony/Component/Mailer/Bridge/Postmark/Transport/PostmarkApiTransport.php

```diff
@@ -143,4 +155,11 @@ private function getEndpoint(): ?string
     {
         return ($this->host ?: self::HOST).($this->port ? ':'.$this->port : '');
     }
+
+    public function setMessageStream(string $messageStream): self
```

Comment:
please add `@return $this` to all methods that `return $this;`

Reply:
Done!